eng/Versions.props

```diff
@@ -245,12 +245,12 @@
     <CommandLineParserVersion>2.3.0</CommandLineParserVersion>
     <FSharpCoreVersion>6.0.0</FSharpCoreVersion>
     <GoogleApiCommonProtosVersion>2.5.0</GoogleApiCommonProtosVersion>
-    <GoogleProtobufVersion>3.18.1</GoogleProtobufVersion>
-    <GrpcAspNetCoreVersion>2.45.0</GrpcAspNetCoreVersion>
-    <GrpcAspNetCoreServerVersion>2.45.0</GrpcAspNetCoreServerVersion>
-    <GrpcAuthVersion>2.45.0</GrpcAuthVersion>
-    <GrpcNetClientVersion>2.45.0</GrpcNetClientVersion>
-    <GrpcToolsVersion>2.45.0</GrpcToolsVersion>
+    <GoogleProtobufVersion>3.21.5</GoogleProtobufVersion>
+    <GrpcAspNetCoreVersion>2.49.0-pre1</GrpcAspNetCoreVersion>
+    <GrpcAspNetCoreServerVersion>2.49.0-pre1</GrpcAspNetCoreServerVersion>
+    <GrpcAuthVersion>2.49.0-pre1</GrpcAuthVersion>
+    <GrpcNetClientVersion>2.49.0-pre1</GrpcNetClientVersion>
+    <GrpcToolsVersion>2.49.0-pre1</GrpcToolsVersion>
```

Comment:
We can't move to prerelease versions w/o messing up the RTM release. Are you sure these will have stable versions before we ship❔

Reply:
Yes. Should be available in 1-2 weeks.

Worst case, we revert.